### PR TITLE
Add mirotalk-bro recipe

### PR DESCRIPTION
**Pull request by miroslavpejic85**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

#### Description of Change

New release - add MiroTalk Live Broadcast to recipes.

MiroTalk WebRTC Live broadcasting involves the real-time transmission of audio, video, and data streams from a broadcaster to multiple viewers. Instead of relying on a centralized server to distribute the stream to viewers, WebRTC enables a direct connection between the broadcaster and each viewer. This approach offers several advantages, such as lower latency, scalability, and reduced infrastructure costs.

Live demo: https://bro.mirotalk.com/
GitHub: https://github.com/miroslavpejic85/mirotalkbro
Note: Unlimited time, unlimited rooms each having a broadcast and many viewers.

Thank you!

### CODE REVIEW

File: recipes/mirotalk-bro/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+	"id": "mirotalk-bro",
```
Comment (vraravam) [inline recipes/mirotalk-bro/package.json:2]: please fix indentation - we prefer 2 spaces

Comment (vraravam) [reply recipes/mirotalk-bro/package.json:2]: these are still using 4 spaces - please convert to 2 spaces (this is also specified/configured in the `.editorconfig` file at the root of the repo.

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by miroslavpejic85** [comment]
This is very strange as per default the [.editorconfig](https://github.com/miroslavpejic85/ferdium-recipes-mirotalk/blob/main/.editorconfig) is configured correctly.

And also on my repo from where I sent the PR [seems ok](https://github.com/miroslavpejic85/ferdium-recipes-mirotalk/blob/main/recipes/mirotalk-bro/package.json) to me.

I missing something? :)

Thank you.

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@all-contributors please add @miroslavpejic85 for code

**Comment by miroslavpejic85** [comment]
Yes, already there, thank you so much guys!

**Pull request merged**